boot.php

```diff
@@ -19,6 +18,16 @@
     });
 }
 
+if (
+    rex_addon::get('yrewrite')->isAvailable() && $this->getConfig('multidomain')
```

Comment:
Ich würde das anders umsetzen. Ich würde gar keine neue Option dafür einführen.
Und dann würde ich es in den oberen Block mit integrieren. Einfach da, wo der User gecheckt wird (Z. 15) vorne dran per OR den Debug-Modus-Check.